### PR TITLE
Logging stack installation scripts

### PR DESCRIPTION
**Pull request by michaelact**
README: https://github.com/GDP-ADMIN/codehub/tree/f/logging-stack/logging-stack

### CODE REVIEW

File: logging-stack/try-localhost.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -e
+
+if [ "$#" -ne 1 ]; then
+  echo "Usage: $0 <service>"
+  echo "Available options: opensearch"
```
Comment (darfisultoni) [inline logging-stack/try-localhost.sh:6]: use constant for this

File: logging-stack/try-localhost.sh [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+SERVICE=$1
+
+if [ "$SERVICE" == "opensearch" ]; then
+  echo "Starting OpenSearch setup..."
+
+  if [ -f ./opensearch/server/standalone-cluster/install.sh ]; then
+    echo "Running install.sh in standalone-cluster..."
+    (cd ./opensearch/server/standalone-cluster && ./install.sh)
+  else
+    echo "install.sh not found in standalone-cluster. Skipping..."
```
Comment (darfisultoni) [inline logging-stack/try-localhost.sh:19]: add info to resolve the issue

File: logging-stack/try-localhost.sh [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  fi
+
+  if [ -f ./opensearch/client/filebeat-agent/install.sh ]; then
+    echo "Running install.sh in filebeat-agent..."
+    (cd ./opensearch/client/filebeat-agent && ./install.sh)
+  else
+    echo "install.sh not found in filebeat-agent. Skipping..."
```
Comment (darfisultoni) [inline logging-stack/try-localhost.sh:26]: add info to resolve the issue

File: logging-stack/try-localhost.sh
Comment (darfisultoni) [inline logging-stack/try-localhost.sh:1]: ini maksudnya quickstart example ya?

File: logging-stack/sample-app/README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+### Overview
+
+This is a sample application that demonstrates how to meet the [Application Requirements](../README.md#application-requirements).
+
+### Prerequisites
+
+Ensure you have the following tools installed on your system:
+
+- Install [Git](https://git-scm.com/downloads) - for version control and cloning the repository.
+- Install [Rancher Desktop](https://rancherdesktop.io/) or [Docker](https://docs.docker.com/engine/install/) - for containerizing and running the application locally.
```
Comment (darfisultoni) [inline logging-stack/sample-app/README.md:12]: pakai nix :moyai:

File: logging-stack/try-localhost.sh
Comment (michaelact) [reply logging-stack/try-localhost.sh:1]: Iya benar

File: logging-stack/try-localhost.sh (hunk 1 above)
Comment (michaelact) [reply logging-stack/try-localhost.sh:6]: Constant maksudnya gimana tu mas, enum kah, udh di strict ke "opensearch" sih
```
if [ "$SERVICE" == "opensearch" ]; then
```

**Review by darfisultoni — APPROVED** [review]

**Review by muhammad-asn — APPROVED** [review]

**Pull request merged**